packages/web-components/src/image/image.ts

```diff
@@ -0,0 +1,54 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+import { ImageFit, ImageShape } from './image.options.js';
+
+/**
+ * The base class used for constucting a fluent image custom element
+ * @public
+ */
+export class Image extends FASTElement {
+  /**
+   * Image layout
+   *
+   * @public
+   * @remarks
+   * HTML attribute: block.
+   */
+  @attr({ mode: 'boolean' })
+  public block?: boolean;
+  /**
+   * Image border
+   *
+   * @public
+   * @remarks
+   * HTML attribute: border.
+   */
+  @attr({ mode: 'boolean' })
+  public bordered?: boolean;
+  /**
+   * Image shadow
+   *
+   * @public
+   * @remarks
+   * HTML attribute: shadow.
+   */
+  @attr({ mode: 'boolean' })
+  public shadow?: boolean;
+  /**
+   * Image fit
+   *
+   * @public
+   * @remarks
+   * HTML attribute: fit.
+   */
+  @attr
+  public fit: ImageFit;
```

Comment:
with @miroslavstastny's incoming change for strict mode we'll want this and shape as optional: `fit?: ImageFit;` as an example